test/ConverterTest.js

```diff
@@ -53,9 +53,9 @@ describe('Converter', function() {
 
 		it('should convert little bit more complex markdown to react elements', function() {
 			var converter = new Converter();
-			var reactElement = converter.convert('# Hello\n\nMore content...');
```

Comment:
Hey! I'm interested in getting this package upgraded too. 

Each of the three dots `...` should probably be replaced with the actual ellipsis character `…` based upon this new feature: https://github.com/showdownjs/showdown/releases/tag/1.8.0